### PR TITLE
[RFC] vim-patch:7.4.575

### PR DESCRIPTION
**Pull request by fmoralesc**
```
updated for version 7.4.575
Problem:    Unicode character properties are outdated.
Solution:   Update the tables with the latest version.
```

https://code.google.com/p/vim/source/detail?r=v7-4-575

### CODE REVIEW

**Comment by oni-link** [comment]
#2159 would automate the updating of these tables.

**Comment by fmoralesc** [comment]
@oni-link Bummer, I didn't notice that PR when starting this one.

File: src/nvim/mbyte.c [hunk 1]
```diff
@@ -1154,22 +1152,14 @@ int utf_char2cells(int c)
     {0x2b55, 0x2b59},
     {0x3248, 0x324f},
     {0xe000, 0xf8ff},
+    {0xfe00, 0xfe0f},
     {0xfffd, 0xfffd},
     {0x1f100, 0x1f10a},
     {0x1f110, 0x1f12d},
-    {0x1f131, 0x1f131},
+    {0x1f130, 0x1f169},
     {0x1f13d, 0x1f13d},
```
Comment (oni-link) [inline src/nvim/mbyte.c]: This line should also be removed.

File: src/nvim/mbyte.c [hunk 2]
```diff
@@ -1932,7 +1923,7 @@ bool utf_iscomposing(int c)
     {0x1082, 0x108d},
     {0x108f, 0x108f},
     {0x109a, 0x109d},
-    {0x135f, 0x135f},
+    {0x135d, 0x135f},
```
Comment (oni-link) [inline src/nvim/mbyte.c]: Cannot comment on the right line:
Missing change `{0x17b6, ...}` -> `{0x17b4, ...}`

**Comment by oni-link** [comment]
Found some differences while comparing the tables with the generated tables from #2159 (and vim-patch).

**Comment by fmoralesc** [comment]
Thanks for taking as look, I'll review it later.

**Comment by fmoralesc** [comment]
OK, Updated it with the changes. If this has differences to what #2159 gives, I think they can be fixed once that gets merged.

**Comment by oni-link** [comment]
LGTM.

**Comment by justinmk** [comment]
Merged, thank you both!

**Pull request closed without merging**